### DISCUSSION TITLE
A command to remove all shops from a chunk would be a nice feature

### DISCUSSION THREAD

**Discussion by jwkerr** [PotatoCraft-Studio/QuickShop-Reremake#144] [Ideas]
Would be useful for server staff, especially if you could choose to remove by player or all of them at once.

**Comment by carmeops** [comment]
the same command for adding/removing staff could be nice too

basically, add at the end of a command "chunk" for all of your shops in the chunk or "all" for all of your shops, period

i got dozens of chest shops, and adding a new staff member who can refill them for me is a pain in the butt
